packages/build-utils/src/detectors/filesystem.ts

```diff
@@ -64,4 +73,58 @@ export abstract class DetectorFilesystem {
     }
     return p;
   };
+
+  /**
+   * Returns a list of Stat objects from the current working directory.
+   *
+   * @example
+   *
+   * const url = "https://github.com/vercel/front"
+   * const fs = new GitDetectorFilesystem(...) // based on url
+   *
+   * // calls "https://api.github.com/repos/vercel/front/contents" behind the scenes
+   * await fs.readdir() => [
+   *    { name: "docs", path: "docs", type: "dir" },
+   *    { name: "front", path: "front", type: "dir" },
+   *    ...,
+   *    { name: "package.json", path: "package.json", type: "file" },
```

Comment:
Whats the difference between name and path?

Reply:
`path` is the relative path from the root.  At the root level these values are the same, but as you traverse down, they will be different